plugins/Table2/scripts/Table.js

```diff
@@ -0,0 +1,37 @@
+const { ContentTab, openTab } = await require('@bridge/tab')
+const { create } = await require('@bridge/sidebar')
+const { Table2 } = await require('@bridge/ui')
+const { register, addTabActions } = await require('@bridge/tab-actions')
+const { setup } = await require('@bridge/com-mojang')
+
+
+class Table2Tab extends ContentTab {
+    type = 'Table2Tab'
+    component = Table2
+
+	async setup() {
+		await new Promise((resolve) => setup.once(async () => {
+            addTabActions(this)
+            resolve()
+        }))
+		await super.setup()
+	}
+	async isFor() { return false }
+	get icon() { return 'mdi-google-spreadsheet' }
+	get iconColor() { return 'warning' }
+	get name() { return 'Table2' }
+}
+
+register({ isFor(tab) { return tab.type === 'Table2Tab' } })
```

Comment:
Same here, make sure to remove the `register` call if you don't actually register a new tab action for your tab.